test/parallel/test-fs-promises.js

```diff
@@ -113,115 +116,134 @@ async function getHandle(dest) {
-      await handle.chown(process.getuid(), process.getgid());
+    // async read from file
+    {
+      const buf = Buffer.from('hello fsPromises');
+      const bufLen = buf.length;
+      await handle.write(buf);
+      const ret = await handle.read(Buffer.alloc(bufLen), 0, bufLen, 0);
+      assert.strictEqual(ret.bytesRead, bufLen);
+      assert.deepStrictEqual(ret.buffer, buf);
     }
 
-    assert.rejects(
-      async () => {
-        await chown(dest, 1, -1);
-      },
-      {
-        code: 'ERR_OUT_OF_RANGE',
-        name: 'RangeError [ERR_OUT_OF_RANGE]',
-        message: 'The value of "gid" is out of range. ' +
-                 'It must be >= 0 && < 4294967296. Received -1'
-      });
-
-    assert.rejects(
-      async () => {
-        await handle.chown(1, -1);
-      },
-      {
-        code: 'ERR_OUT_OF_RANGE',
-        name: 'RangeError [ERR_OUT_OF_RANGE]',
-        message: 'The value of "gid" is out of range. ' +
-                  'It must be >= 0 && < 4294967296. Received -1'
-      });
-
-    await utimes(dest, new Date(), new Date());
-
-    try {
-      await handle.utimes(new Date(), new Date());
-    } catch (err) {
-      // Some systems do not have futimes. If there is an error,
-      // expect it to be ENOSYS
-      common.expectsError({
-        code: 'ENOSYS',
-        type: Error
-      })(err);
+    // async read from file handle
+    {
+      const buf = Buffer.from('hello FileHandle');
+      const bufLen = buf.length;
+      await handle.write(buf, 0, bufLen, 0);
+      const ret = await handle.read(Buffer.alloc(bufLen), 0, bufLen, 0);
+      assert.strictEqual(ret.bytesRead, bufLen);
+      assert.deepStrictEqual(ret.buffer, buf);
+      await truncate(dest, 5);
+      assert.deepStrictEqual((await readFile(dest)).toString(), 'hello');
     }
 
-    await handle.close();
+    // invalid change of ownership
+    {
+      await chmod(dest, 0o666);
+      await handle.chmod(0o666);
 
-    const newPath = path.resolve(tmpDir, 'baz2.js');
-    await rename(dest, newPath);
-    stats = await stat(newPath);
-    verifyStatObject(stats);
+      await chmod(dest, (0o10777));
+      await handle.chmod(0o10777);
 
-    if (common.canCreateSymLink()) {
-      const newLink = path.resolve(tmpDir, 'baz3.js');
-      await symlink(newPath, newLink);
       if (!common.isWindows) {
-        await lchown(newLink, process.getuid(), process.getgid());
+        await chown(dest, process.getuid(), process.getgid());
+        await handle.chown(process.getuid(), process.getgid());
       }
-      stats = await lstat(newLink);
-      verifyStatObject(stats);
 
-      assert.strictEqual(newPath.toLowerCase(),
-                         (await realpath(newLink)).toLowerCase());
-      assert.strictEqual(newPath.toLowerCase(),
-                         (await readlink(newLink)).toLowerCase());
+      assert.rejects(
+        async () => {
+          await chown(dest, 1, -1);
+        },
+        {
```

Comment:
The error object on line 158 and 169 is same and can be stored in a variable instead:
```jason
{
  code: 'ERR_OUT_OF_RANGE',
  name: 'RangeError [ERR_OUT_OF_RANGE]',
  message: 'The value of "gid" is out of range. ' +
    'It must be >= 0 && < 4294967296. Received -1'
}
```